op-service/sources/l1_beacon_client.go

```diff
@@ -120,10 +120,16 @@ func (cl *BeaconHTTPClient) BeaconBlobSideCars(ctx context.Context, fetchAllSide
 	reqPath := path.Join(sidecarsMethodPrefix, strconv.FormatUint(slot, 10))
 	var reqQuery url.Values
 	if !fetchAllSidecars {
-		reqQuery = url.Values{}
+		indexString := ""
```

Comment:
We can also use https://pkg.go.dev/strings#Join here

Reply:
Updated in https://github.com/ethereum-optimism/optimism/pull/10540/commits/6803524c9f3948cb397bdac4badb36a574f1cc56